### PR TITLE
Improve handling of repository updates across version, add 'announce:'

### PR DESCRIPTION
**Pull request by AltGr**

### CODE REVIEW

File: doc/pages/Manual.md [hunk 1]
```diff
@@ -577,6 +577,9 @@ some specifics of the repository. It has the following optional fields:
   similar [config field](#configfield-archive-mirrors), except a path without a
   `protocol://` prefix is accepted and will be considered relative to the
   repository root (e.g. `cache/`).
+* <a id="repofield-announce">`redirect: [ <string> { <filter> } ... ]`</a>:
```
Comment (hannesm) [inline doc/pages/Manual.md]: `redirect:` -> `announce:`

**Comment by avsm** [comment]
Great feature! Would this be useful to slightly generalise to add an `motd` on `opam update`? That would let us signpost breaking changes to users more easily for more than just first init.

**Comment by AltGr** [comment]
That's what it is, it will be displayed on every update! There are filters too, so we could add variables `{init}`, `{update}`; but is there a need to ?

File: doc/pages/Manual.md (hunk 1 above)
Comment (AltGr) [inline doc/pages/Manual.md]: indeed! Thanks for spotting this

**Pull request merged**